test/integration/components/testserver_1.17/go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/open-telemetry/opentelemetry-ebpf-instrumentation/testserver_1.17
 
-go 1.17
+go 1.24
```

Comment:
We need to keep this one at 1.17, otherwise the alternative is to completely remove the test. This was added to ensure we still work with the oldest version of Go we technically support. We've broken this in the past so the test was added.